ONSdigital/eq-author-app
# EAR 1948 Option values for radio/cb/select answers

### What is the context of this PR?

> Add the capability to specify values for each option in a radio/checkbox/select answer types.

### How to review

- Create survey with radio, checkbox (with mutually exclusive option) and a select question
- Check QCodes and Values page that extra column is visible
- Check Option Values can be added to the options fro radio, checkbox and select answers
- Check the schema that these values have been saved
- Check that validation errors show when values are missing and disappear when values have been added.

### What to do after everything is green

1. - [ ] Bring this branch up-to-date with Origin/Master
2. - [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. - [ ] Click the **Merge** button on this pull request
4. - [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. - [ ] Move the Jira ticket for this task into the next stage of the process
